README.md

```diff
@@ -184,6 +184,24 @@ If the execution is specified (does not run by default), the `containerize-depen
 	</plugin>
 ```
 
+
+To update the Dockerfile with containerize dependencies logic, you need to specify the Dockerfile location by adding the `dockerfile` tag under `configuration` and, also, to insert both the `#HABUSHU_BUILDER_STAGE` and `#HABUSHU_FINAL_STAGE` tags in the Dockerfile in the preferred location. 
```

Comment:
```suggestion
To update the `Dockerfile` with logic to leverage the monorepo dependencies, specify the `Dockerfile` location by adding the `dockerfile` tag under `configuration` and to insert both the `#HABUSHU_BUILDER_STAGE` and `#HABUSHU_FINAL_STAGE` tags in the `Dockerfile` in the preferred location. 
```

Reply:
I'd prefer "logic to containerize the python application"